CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# 0.3.0
+
+This version roughly corresponds with all the changes made to the Temporal polyfill as of the October 2021 TC39 Plenary meeting.
+
+Breaking changes:
+
+- Timezones now require a `getOffsetNanosecondsFor` method, and no longer fall back to the intrinsic definition (previously provided by `Temporal.Timezone#getOffsetNanosecondsFor`) if not provided. ([08346dc5])
+- Disallow Z designators when parsing strings for Plain Temporal types ([f3f8a994], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1874))
+- Allow ISO strings with "Z" + a bracketed IANA name ([70bd9898], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1749))
+- Emit ES2020 builds for newer browsers, and emit ES5 for older browsers. ([2331468d], [9e95c62b])
+- Temporal.Duration constructor will now throw if given a non-integer ([9df5d068], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1872))
+- Remove support for sub-minute offsets in ISO strings ([766e5037], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1871), [Spec PR](https://github.com/tc39/proposal-temporal/pull/1862))
+- Throw TypeError on missing options from Duration.total ([4ec075f0], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1720))
+- Reject non-integer Duration fields in Duration.with() ([e6b2488d], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1735))
+- Ensure an Object is returned from calendar.mergeFields() ([4e63f25f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1719))
+
+Bug fixes:
+
+- Fix GetFormatterParts for Firefox Nightly ([47f9132f])
+- Fix TS types of RoundTo and TotalOf ([3008a670])
+- Fix crash setting `day` outside current JPN era ([6d3588c3], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1807))
+- Copy options object for PlainYearMonth.{add,subtract} and InterpretTemporalDateTimeFields to prevent user-modified objects from interfering with later operations. ([bafa1bdf], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1748))
+- Validate input to Calendar.prototype.fields ([7ebc700e], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1750))
+- Stop observably calling into `getPossibleInstantsFor` in `InterpretISODateTimeOffset` ([5448e59f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1688))
+- Call `CalendarEquals` correctly ([07ea694e], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1858))
+- Fix arithmetic issues when using non-ISO months ([079a3325], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1761))
+
+New changes:
```

Comment:
Should we call this "Non-breaking changes" for symmetry, and move this section below the "Breaking changes" section?

Reply:
Maybe call it "New features", but keep it where it is?

 Personally, I think "breaking changes > bug fixes > new stuff that isn't a bugfix > misc documentation for maintainers" is the right order.

Reply:
> Maybe call it "New features", but keep it where it is?
For a regular OSS package I'd agree, but as a Stage 3 polyfill claiming that there are new "features" seems like a bad idea because the idea of Stage 3 is that there won't be any new features!  I'd prefer to call them "Non-breaking changes" or some other wording that doesn't make it sound like the proposal is a moving target.

> Personally, I think "breaking changes > bug fixes > new stuff that isn't a bugfix > misc documentation for maintainers" is the right order.

Sure that's OK with me.

Reply:
I suppose in terms of functionality there's no new features, though accepting strings to round and total sure seemed like a "feature" to me 😆 

Non-breaking definitely makes more sense with your explanation, so let's use that.